lib/plugins/browsertime/analyzer.js

```diff
@@ -73,7 +73,11 @@ module.exports = {
     let engine = new browsertime.Engine(btOptions);
     log.info('Starting %s for analysing %s %s time(s)', btOptions.browser, url, btOptions.iterations);
     return engine.start()
-      .then(() => engine.run(url, scriptsByCategory))
+      .then((result,error) => engine.run(url, scriptsByCategory))
+      .catch((error) => {
+        process.exitCode = 1;
+        process.exit();
```

Comment:
Ah I see what you mean let's focus on fixing it in browsertime via a result json then. :-)